src/SourceGenerators/Uno.UI.SourceGenerators.netcore.Tests/XamlCodeGeneratorTests/TestOutput/When_ObservableProperty_AttributeDoesNotExists/MainPage_d6cd66944958ced0c513e0a04797b51d.cs

```diff
@@ -0,0 +1,265 @@
+﻿// <autogenerated />
+#pragma warning disable CS0114
+#pragma warning disable CS0108
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using Uno.UI;
+using Uno.UI.Xaml;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Documents;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Media.Animation;
+using Windows.UI.Xaml.Shapes;
+using Windows.UI.Text;
+using Uno.Extensions;
+using Uno;
+using Uno.UI.Helpers.Xaml;
+using MyProject;
+
+#if __ANDROID__
+using _View = Android.Views.View;
+#elif __IOS__
+using _View = UIKit.UIView;
+#elif __MACOS__
+using _View = AppKit.NSView;
+#elif UNO_REFERENCE_API || NET461
+using _View = Windows.UI.Xaml.UIElement;
+#endif
+
+namespace TestRepro
+{
+	partial class MainPage : global::Windows.UI.Xaml.Controls.Page
+	{
+		[global::System.ComponentModel.EditorBrowsable(global::System.ComponentModel.EditorBrowsableState.Never)]
+		private const string __baseUri_prefix_MainPage_d6cd66944958ced0c513e0a04797b51d = "ms-appx:///TestProject/";
+		[global::System.ComponentModel.EditorBrowsable(global::System.ComponentModel.EditorBrowsableState.Never)]
+		private const string __baseUri_MainPage_d6cd66944958ced0c513e0a04797b51d = "ms-appx:///TestProject/";
+				global::Windows.UI.Xaml.NameScope __nameScope = new global::Windows.UI.Xaml.NameScope();
+		private void InitializeComponent()
+		{
+			InitializeComponent_59289A66();
+		}
+		private void InitializeComponent_59289A66()
+		{
+			NameScope.SetNameScope(this, __nameScope);
+			var __that = this;
+			base.IsParsing = true;
+			// Source 0\MainPage.xaml (Line 1:2)
+			base.Content = 
+			new global::Windows.UI.Xaml.Controls.StackPanel
+			{
+				IsParsing = true,
+				// Source 0\MainPage.xaml (Line 6:3)
+				Children = 
+				{
+					new global::Windows.UI.Xaml.Controls.TextBlock
+					{
+						IsParsing = true,
+						// Source 0\MainPage.xaml (Line 7:4)
+					}
+					.MainPage_d6cd66944958ced0c513e0a04797b51d_XamlApply((MainPage_d6cd66944958ced0c513e0a04797b51dXamlApplyExtensions.XamlApplyHandler0)(c0 => 
+					{
+						/* _isTopLevelDictionary:False */
+						__that._component_0 = c0;
+						c0.SetBinding(
+							global::Windows.UI.Xaml.Controls.TextBlock.TextProperty,
+							new Windows.UI.Xaml.Data.Binding()
+							{
+								Mode = BindingMode.OneTime,
+							}
+								.BindingApply(___b =>  /*defaultBindModeOneTime ViewModel.Name.ToUpper()*/ global::Uno.UI.Xaml.BindingHelper.SetBindingXBindProvider(___b, __that, ___ctx => ___ctx is global::TestRepro.MainPage ___tctx ? ((true, ___tctx.ViewModel.Name.ToUpper())) : (false, default), null ))
```

Comment:
Note: The x:Bind nullability rewriter doesn't account for source-generated members.

We could try to add that support for CommunityToolkit.Mvvm.